### ISSUE TITLE
Does not work well in Virtual Machine

### ISSUE THREAD

**Issue by brenns10** [brenns10/alien-iso#1]
Plymouth stays in console mode and falls back to a crappy splash screen, rather than the NOSTROMO splash. Also, it appears that cool-retro-term relies on hardware for some of its neat looking effects, and so it lags quite a bit on a VM. Tested qemu and Virtualbox. Potentially installing virtualbox guest additions would help.

**Comment by xeon-zolt** [comment]
for plymouth try increasing screen resolution of VBox

**Comment by brenns10** [comment]
Yeah, I saw that the guest additions didn't really help. I'll see about increasing the resolution.
